codegen/templates/chart/operator-deployment.yamltmpl

```diff
@@ -95,6 +95,12 @@ spec:
 {{- if [[ $containerVar ]].env }}
         env:
 {{ toYaml [[ $containerVar ]].env | indent 10 }}
+{{- else if [[ $containerVar ]].extraEnvs }}
+        env:
+{{- end }}
+{{- range $name, $item := [[ $containerVar ]].extraEnvs }}
+          - name: {{ $name }}
+{{- $item | toYaml | nindent 12 }}
```

Comment:
Why is this list `else if` with `env`? Isn't this supposed to be an append style operation

Reply:
It is to ensure the `env:` top level key is always set if either of `env` or `extraEnvs` are set, but ensure `env:` doesn't get set twice if both are set.